src/Microsoft.Azure.WebJobs.Host/Bindings/BindingProviders/FluentBindingProviderExtensions.cs

```diff
@@ -11,16 +11,14 @@ namespace Microsoft.Azure.WebJobs.Host
 {    
     // Internal Extension methods for setting backwards compatibility hooks on certain bindings. 
     // This keeps the hooks out of the public surface. 
-    internal static class FluidBindingProviderExtensions
+    internal static class FluentBindingProviderExtensions
```

Comment:
This is safe to do because you're going to send a corresponding PR removing the hook dependency from the Bot binding? Otherwise, this would break them.